client/src/pages/SignUp/SignUp.tsx

```diff
@@ -40,14 +40,14 @@ export default function Register(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid className={classes.paper} component={Paper} square>
         <Box className={classes.authWrapper}>
           <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
+          <Box className={classes.border} width="100%" maxWidth={450} p={3} alignSelf="center">
```

Comment:
As opposed to using a class to style a solid border here, maybe consider the usage of a MUI `<Paper>` component?

Reply:
Does it matter if it is a <Paper> component or using componet={Paper}?

Reply:
If you use a `<Paper>` component, you can specify additional props (for eg - elevation)